src/main/java/space/obminyashka/items_exchange/controller/UserController.java

```diff
@@ -105,6 +106,24 @@ public Page<AdvertisementTitleDto> getFavoriteAdvertisements(
         return advService.findAllFavorite(authentication.getName(), PageRequest.of(page, size));
     }
 
+    @PreAuthorize("hasAnyRole('USER', 'ADMIN', 'MODERATOR')")
+    @PostMapping(value = ApiKey.USER_ADD_MY_FAVORITE)
+    @Operation(summary = "Add user's favorite advertisement")
+    @ApiResponses(value = {
+            @ApiResponse(responseCode = "202", description = "ACCEPTED"),
+            @ApiResponse(responseCode = "401", description = "UNAUTHORIZED"),
+            @ApiResponse(responseCode = "404", description = "NOT_FOUND")})
+    @ResponseStatus(HttpStatus.ACCEPTED)
+    public void addFavoriteAdvertisement(
+            @Parameter(hidden = true) Authentication authentication,
+            @Parameter(name = "advertisementId", description = "Id of advertisement for to add an favorite adv")
+            @PathVariable UUID advertisementId) {
+        if (!advService.existById(advertisementId)) {
+            throw new EntityIdNotFoundException(getMessageSource(ResponseMessagesHandler.ExceptionMessage.ADVERTISEMENT_NOT_EXISTED_ID));
```

Comment:
Would you mind applying here static import for the exception message key?

Reply:
made